foreach2.js

```diff
@@ -12,3 +12,6 @@
 
 // your code here
 
+Object.values(foods).forEach(val => {
+  console.log(val);
+});
```

Comment:
hm... you seem to be `console.log`ging the objects as they already exist in the array. You should be printing strings to the console, not objects. give this another look if you have a chance, see if you can get the output described in lines 3-5.